### ISSUE TITLE
Screen may not always shift enough to keep focus on screen

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecules-and-light#206]
For https://github.com/phetsims/QA/issues/419. Win 10 Chrome.
When using keyboard nav when zoomed in, the screen pans to keep the focused element on screen.  However, it is possible for this to not occur.  If the element is technically on screen, but the relevant part isn't, the panning doesn't happen and this may cause pedagogical problems, or even make it difficult to tell keyboard nav is active.
Steps:
1. Refresh the sim so that keyboard nav focus is reset.
2. Zoom in and pan so that the light emitter is on the screen, but the control is not
3. Hit tab to start keyboard nav and focus on the emitter control
4. Hit tab again to move focus to the next element and see that the screen does still follow focus
5. Shift tab to see where the screen should have been during (3)

Video:
https://drive.google.com/file/d/1LIZTzGfmobWOrGefDy7-qiDf4jYkagpL/view?usp=sharing

**Comment by KatieWoe** [comment]
This can also be a problem when shifting between options.  Specifically, the default position for the screen when zoomed in and focused on the control for type of light is to start on the left of the panel and cut off the right side.  If you move the selecting focus to Ultraviolet, the screen does not pan to fully show the option and it can be unclear what you are selecting.

**Comment by jessegreenberg** [comment]
Thanks for pointing out. The behavior is "if the focused item is completely outside of the visible area, we pan to the focused item". I think that both of the cases you pointed out might be improved if it we change it to be "If any portion of the focused item is outside the visible area, pan so that the entire item is within visible bounds.

**Comment by jessegreenberg** [comment]
I went with https://github.com/phetsims/molecules-and-light/issues/206#issuecomment-530434145. This fixes the issue pointed out by @KatieWoe. The result feels a little more "agressive", the sim moves more sometimes. For example, if something is mostly in view but just slightly out of the window, we will still move the whole sim so that the node gets centered.

@KatieWoe do you think this is better in this version? https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-customZoom.8/phet/molecules-and-light_en_phet.html

(buttons aren't working quite right here, please ignore that)
